eng/pipelines/templates/stages/vmr-cross-build.yml

```diff
@@ -89,6 +89,8 @@ stages:
       withPreviousSDK: false             # 🚫
       crossRootFs: '/crossrootfs/x64'    # 📝
       targetRid: 'linux-x64'             # 📝
```

Comment:
Do we still need to pass targetRid in? I see it's only use here https://github.com/dotnet/installer/blob/e2338d7a79630ef3a488f03bde9497c4a110fb6e/eng/pipelines/templates/jobs/vmr-build.yml#L296-L310

Reply:
I left it alone because of that usage. I guess we could have a RunTests property to stop using inferred behaviour (i.e. "should I run tests" is a separate question from "do I have a target RID" and should be gated on a relevant property)